### PR TITLE
Filter resources triggered by namespace

### PR DESCRIPTION
**Pull request by inteon**
When a namespace changes, we only want to re-reconcile all bundles that have that namespace as a target.

### CODE REVIEW

File: pkg/bundle/bundle.go [hunk 1]
```diff
@@ -244,3 +244,14 @@ func (b *bundle) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result,
 
 	return ctrl.Result{}, b.client.Status().Update(ctx, &bundle)
 }
+
+func (b *bundle) bundleTargetNamespaceSelector(bundleObj *trustapi.Bundle) (labels.Selector, error) {
+	namespaceSelector := labels.Everything()
```
Comment (erikgb) [inline pkg/bundle/bundle.go:249]: Nit: Is this variable needed? Why not just inline it?

Comment (inteon) [reply pkg/bundle/bundle.go]: I updated the code, also made some more changes. PTAL

**Review by erikgb — APPROVED** [review]
/lgtm

**Pull request merged**